.github/ISSUE_TEMPLATE/bug_issue.yml

```diff
@@ -0,0 +1,33 @@
+name: 🐛 Bug
```

Comment:
It looks good but it should be simple enough I believe, required field should not be true!

Could you checkout this simple template, I like [this](https://github.com/CircuitVerse/CircuitVerse/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) one.

Reply:
Apply this in other templates as well!

Reply:
Hey Vedant, thanks for the feedback, I have performed some changes in my [fork](https://github.com/itsEobard2025/certificate-generator). Could you test it out(making dummy issues/PRs) and tell me if it requires further enhancements ?